squad/core/management/commands/migrate_attachments.py

```diff
@@ -65,20 +64,38 @@ def add_arguments(self, parser):
         parser.add_argument(
             '--show-progress',
             action='store_true',
-            help='Prints out one dot every 1000 (one thousand) testruns processed'
+            help='Prints out one dot every 100 (one thousand) testruns processed'
+        )
+        parser.add_argument(
+            '--num-threads',
+            type=int,
+            default=2,
+            help='Number of simultaneous parallel threads to work'
+        )
+        parser.add_argument(
+            '--groups',
+            default='all',
+            help='List of groups slugs separated by ","'
         )
 
     def handle(self, *args, **options):
         show_progress = options['show_progress']
+        num_threads = options['num_threads']
+        groups = options['groups']
 
         null_storage_files = Q(tests_file_storage__isnull=True) | Q(metrics_file_storage__isnull=True) | Q(log_file_storage__isnull=True)
-        testrun_ids = TestRun.objects.filter(null_storage_files).order_by('id').values_list('id', flat=True)
+        queryset = TestRun.objects.filter(null_storage_files)
+
+        if groups != 'all':
```

Comment:
'all' is a valid group name. Maybe default should be empty list? You could use nargs=+ to specify that you're after a list.

Reply:
Oh, I didn't think of it, let me fix it

Reply:
Done, I've just make `default=''`